### PR TITLE
fix: added support for winrm_password for windows custom images

### PR DESCRIPTION
**Pull request by ujjwal-ibm**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -225,6 +225,7 @@ ssh_port | int | Optional | The port that SSH will be available on. Defaults to
 ssh_timeout | string | Optional | The time to wait for SSH to become available before timing out. The format of this value is a duration such as "5s" or "5m".
 ***Windows Communicator Variables*** |
 winrm_username | string | Optional | The username to use to connect to WinRM.
+winrm_login_password | string | Optional | The password to use to connect to WinRM. (For custom windows images)
```
Comment (astha-jain) [inline README.md:228]: I would recommend `winrm_password` keyword in parity with existing `winrm_username`

File: builder/ibmcloud/vpc/config.go [hunk 2]
```diff
@@ -45,6 +45,8 @@ type Config struct {
 	VSIUserDataFile           string `mapstructure:"vsi_user_data_file"`
 	VSIUserDataString         string `mapstructure:"vsi_user_data"`
 
+	WinRMLoginPassword string `mapstructure:"winrm_login_password"`
+
```
Comment (astha-jain) [inline builder/ibmcloud/vpc/config.go:49]: same

**Review by astha-jain — COMMENTED** [review]
Functionality works, Changes suggested on nomenclature.

**Comment by ujjwal-ibm** [comment]
```
==> ibmcloud-vpc.windows:
==> ibmcloud-vpc.windows: ********************************************************************
==> ibmcloud-vpc.windows: * Thank you for using IBM Cloud Packer Plugin - VPC Infrastructure *
==> ibmcloud-vpc.windows: ********************************************************************
Build 'ibmcloud-vpc.windows' finished after 33 minutes 14 seconds.

==> Wait completed after 33 minutes 14 seconds

==> Builds finished. The artifacts of successful builds are:
--> ibmcloud-vpc.windows: Image Name: packer-20240826163654 || Image ID: r006-092b6ee2-06b9-41cc-ab98-5f6739ab3cf9
```
tested. lgtm

**Review by astha-jain — APPROVED** [review]

**Pull request merged**